rfc/src/rfcs/0009-function-contracts.md

```diff
@@ -706,6 +706,22 @@ structures however are ubiquitous and users can avoid the unsoundness with
 relative confidence by overprovisioning (generating inputs that are several
 times larger than what they expect the function will touch).
 
+### Interior Mutability
```

Comment:
I don't know if this should be here. This is not related to contracts. It may be more interesting to put something like this in our wiki. Maybe in the [limitations](https://model-checking.github.io/kani/limitations.html) section.